tests/test_rank.py

```diff
@@ -388,3 +389,48 @@ def test_unsupported_method_for_adjust_pred_probs():
         method = "confidence_weighted_entropy"
 
         _ = rank.get_label_quality_scores(labels, pred_probs, adjust_pred_probs=True, method=method)
+
+
+def test_get_knn_distance_ood_scores():
+
+    X_train = data["X_train"]
+    X_test = data["X_test"]
+
+    # Create OOD datapoint
+    X_ood = np.array([[999999999.0, 999999999.0]])
+
+    # Add OOD datapoint to X_test
+    X_test_with_ood = np.vstack([X_test, X_ood])
+
+    # Fit nearest neighbors on X_train
+    nbrs = NearestNeighbors(n_neighbors=5, algorithm="ball_tree").fit(X_train)
```

Comment:
why ball_tree instead of auto?

Reply:
No good reason, I just took that from one of the sklearn docs. I'll remove for brevity so the default will be "auto".